src/main/zlatopolsky/chapter4/Ch1N4_106.java

```diff
@@ -0,0 +1,29 @@
+package zlatopolsky.chapter4;
+
+public class Ch1N4_106 {
+
+    public String season(int month) {
+        String season = "";
```

Comment:
// I think you can remove variable String season and will do return "winter" etc.

Reply:
i'm delete case 1, 2, 12